karma.conf.js

```diff
@@ -37,8 +37,13 @@ module.exports = (config) => {
                     },
                 },
             },
-            browserDisconnectTolerance: 2,
-            browserNoActivityTimeout: 60000,
+            // our ci uses a 2 core 4 gb ram instance to run tests
+            // higher concurrency reduces reliability of browser timeout
+            // without a significant effect on testing speed
+            concurrency: 1,
+            // this timeout is primarily significant during browser startup
+            // after browser startup we shouldn't be coming close to this limit
+            browserNoActivityTimeout: 30000,
```

Comment:
This is the default value, if we want to keep it here for documentation concerns, please notate that.

Reply:
I'd prefer to keep this value here for multiple reasons. Aside from documentation , I don't want this value changing under us based on upstream changes (i.e. `karma` or `@open-wc/testing-karma` updates).

Reply:
👍 All great reason. Just want to save the next person a half cycle of brain, thanks!